src/util/util_fps_limiter.h

```diff
@@ -53,34 +53,34 @@ namespace dxvk {
      * \returns \c true if the target frame rate is non-zero.
      */
     bool isEnabled() const {
-      return m_targetInterval != NtTimerDuration::zero();
+      return m_targetInterval != NativeTimerDuration::zero();
     }
 
   private:
 
     using TimePoint = dxvk::high_resolution_clock::time_point;
 
-    using NtTimerDuration = std::chrono::duration<int64_t, std::ratio<1, 10000000>>;
+    using NativeTimerDuration = std::chrono::duration<int64_t, std::ratio<1, 10000000>>;
```

Comment:
why not just `TimerDuration`? "Native" doesn't immediately mean anything.

Reply:
Sure